openmc/stats/univariate.py

```diff
@@ -239,7 +239,7 @@ def to_xml_element(self, element_name):
 
 
 class Watt(Univariate):
-    r"""Watt fission energy spectrum.
+    """Watt fission energy spectrum.
```

Comment:
The `r` raw prefix is needed because otherwise things like `\s` will be interpreted as a single character.